src/plugins/kibana/public/management/index.js

```diff
@@ -42,6 +42,16 @@ uiModules
       if ($scope.section) {
         $scope.section.items.forEach(item => {
           item.class = `#${$location.path()}`.indexOf(item.url) > -1 ? 'active' : undefined;
+          // Disable link if path is not set
+          if (!item.path) {
+            delete item.url;
+          }
+          item.items.forEach(childItem => {
+            // Disable link if path is not set
+            if (!childItem.path) {
+              delete childItem.url;
```

Comment:
I guess what I'm trying to say is that I'd rather have an explicit call to a helper function like `setUrl` than a "magic" call to a setter which would automatically happen when the `assign` statement attempts to execute `this.path = options.path`.